Makefile

```diff
@@ -8,7 +8,7 @@ migrate:
 	bundle exec bin/nailed --migrate
 
 fetch:
-	bundle exec bin/nailed --github && \
+	bundle exec bin/nailed --changes && \
```

Comment:
As discussed, I would propose a future update where we unify `--github`, `--gitlab`, which currently is `--changes`, with `--bugzilla` into one update method, e.g. `--fetch`

Reply:
I propose you open up an issue for that.